cmd/utils.go

```diff
@@ -160,16 +167,25 @@ func parseBuildInfoJSON(jsonFile string) (signatureURL, cryptoKey string, buildP
 
 	// get url of the signature zip of the package
 	var sigURL string
-	for _, val := range sbomData.Packages[0].ExternalRefs {
-		if val.ReferenceCategory == "OTHER" {
-			sigURL = val.ReferenceLocator
+	for _, element := range sbomData.Packages {
```

Comment:
can you add comments to explain what its doing? because there is loop inside if condition which is inside a loop.